client/src/test/java/org/spine3/base/MismatchShould.java

```diff
@@ -33,33 +36,70 @@
     private static final String EXPECTED = "expected";
     private static final String FOUND = "found";
     private static final int VERSION = 0;
+    public static final String DEFAULT_VALUE = "";
 
```

Comment:
Please add tests for all methods to cover this class 100%. Use `run with coverage` in IDEA. 
We need good coverage because it is a framework, and users want to be sure it is reliable.